README.md

```diff
@@ -20,20 +20,12 @@ required version.
 
 If you use this software or code, please cite the following papers:
 
-* Ross K. Meentemeyer, Nik J. Cunniffe, Alex R. Cook, Joao A. N. Filipe,
-  Richard D. Hunter, David M. Rizzo, and Christopher A. Gilligan, 2011.
-  Epidemiological modeling of invasion in heterogeneous landscapes:
-  spread of sudden oak death in California (1990–2030).
-  *Ecosphere* 2:art17.
-  [DOI: 10.1890/ES10-00192.1](https://doi.org/10.1890/ES10-00192.1)
-
-* Tonini, Francesco, Douglas Shoemaker, Anna Petrasova, Brendan Harmon,
-  Vaclav Petras, Richard C. Cobb, Helena Mitasova,
-  and Ross K. Meentemeyer, 2017.
-  Tangible geospatial modeling for collaborative solutions
-  to invasive species management.
-  *Environmental Modelling & Software* 92: 176-188.
-  [DOI: 10.1016/j.envsoft.2017.02.020](https://doi.org/10.1016/j.envsoft.2017.02.020)
+* Jones, Chris, Jones, Shannon, Petrasova, Anna, Petras, Vaclav, Gaydos, Devon, 
+  Skrip, Megan, Yu, Takeuchi, Bigsby, Kevin, and Meentemeyer, Ross, 2021.
```

Comment:
This should probably be "Jones, Chris, Shannon Jones, Anna Petrasova, ..." or "Jones, C., Jones, S., Petrasova, A., ..." but if you don't have it at hand, don't worry about that, I can change it when I do some bigger updates for DOI in citation file etc.